components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2077,6 +2077,101 @@ void RewardsServiceImpl::SaveInlineMediaInfo(
                     std::move(callback)));
 }
 
+void RewardsServiceImpl::UpdateMediaDuration(
+    const uint64_t window_id,
+    const std::string& media_type,
+    const std::string& publisher_key,
+    uint64_t duration) {
+  if (!Connected()) {
+    return;
+  }
+
+  bat_ledger_->UpdateMediaDuration(
+      window_id,
+      media_type,
+      publisher_key,
+      duration);
+}
+
+void RewardsServiceImpl::GetPublisherInfo(
+    const std::string& publisher_key,
+    GetPublisherInfoCallback callback) {
+  if (!Connected()) {
+    return;
+  }
+
+  bat_ledger_->GetPublisherInfo(
+      publisher_key,
+      base::BindOnce(&RewardsServiceImpl::OnPublisherInfo,
+          AsWeakPtr(),
+          std::move(callback)));
+}
+
+void RewardsServiceImpl::OnPublisherInfo(
+    GetPublisherInfoCallback callback,
+    const ledger::Result result,
+    ledger::PublisherInfoPtr info) {
+  const auto result_converted = static_cast<int>(result);
```

Comment:
move this after if so that we convert only if result is LEDGER_OK

Reply:
The `if` statement also returns the converted result, though, or am I missing something there?

Reply:
ah sorry yes, missed that